doc/mgr/dashboard.rst

```diff
@@ -1070,3 +1070,160 @@ and loosely coupled fashion.
 
 .. include:: dashboard_plugins/feature_toggles.inc.rst
 .. include:: dashboard_plugins/debug.inc.rst
+
+
+Troubleshooting the Dashboard
+-----------------------------
+
+Locating the Dashboard
+^^^^^^^^^^^^^^^^^^^^^^
+
+If you are unsure of the location of the Ceph Dashboard, run the following command::
+
+    $ ceph mgr services | jq .dashboard
+    "https://host:port"
+
+The command returns the URL where the Ceph Dashboard is located: ``https://<host>:<port>/``
+
```

Comment:
With regards to the comment from @SchoolGuy : maybe we should add a hint about installing `jq` here?

```
Note that many Ceph command line tools return results in JSON format.
You may have to install the ``jq`` command-line JSON processor utility
on your operating system beforehand.
```

Reply:
Thanks for the text @LenzGr! My idea would be to add a link to the jq website as well:

```
Note: Many Ceph command line tools return results in JSON format.
You may have to install the `jq <https://stedolan.github.io/jq>`_ 
command-line JSON processor utility on your operating system beforehand.
```

Reply:
@s0nea That would indeed be the most superb solution in my eyes!